.github/workflows/test-upload-release-assets.yml

```
@@ -0,0 +1,34 @@
+
+      - name: Upload release assets (dry-run)
+        id: upload
+        uses: ./.github/actions/upload-release-assets
+        with:
+          release-tag: v1.0.0
+          bin-name: test-app
+          dist-dir: dist
+          dry-run: "true"
+
+      - name: Verify outputs
+        run: |
```

actionlint (1.7.9)

property "uploaded_count" is not defined in object type {error-message: string; upload-error: string; uploaded-count: string}

(expression)